packages/web/components/trade-tool/index.tsx

```diff
@@ -23,7 +24,10 @@ export const TradeTool: FunctionComponent<TradeToolProps> = () => {
   return (
     <ClientOnly>
       <div className="relative flex flex-col gap-6 overflow-hidden md:gap-6 md:px-3 md:pb-4 md:pt-4">
-        <SwapToolTabs activeTab={tab} setTab={setTab} />
+        <div className="flex w-full items-center justify-between">
+          <SwapToolTabs activeTab={tab} setTab={setTab} />
+          <OrderTypeSelector />
```

Comment:
```suggestion
          {tab !== SwapToolTab.SWAP && <OrderTypeSelector />}
```

Should be disabled for the Swap tab

Reply:
[aa9f5cb](https://github.com/osmosis-labs/osmosis-frontend/pull/3320/commits/aa9f5cbb3d63fc339174e34f1a5487f93855a1d6)